server/src/server.ts

```diff
@@ -395,3 +401,20 @@ connection.onCodeLens(codeLensRequestHandler(definitionsProvider));
 
 // Listen on the connection
 connection.listen();
+
+/**
+ * Create a request handler for the event `documentFormatting`.
+ * The handler, when receiving a `DocumentFormattingParams` instance will find
+ * will parse the current file and build and send the list of Text Edit
```

Comment:
will find will parse? xD (plus missing comma).
Did you merge two existing sentences or something? xD